### DISCUSSION TITLE
Is the project dead? Does anyone still maintain it?

### DISCUSSION THREAD

**Discussion by tobylibo** [jesec/flood#630] [General]
The date of this latest release project is October 9, 2021.

Today is January 28, 2023.

It's been almost a year and a half.

Aren't there any new releases to come? I can't wait. (*￣︶￣)

**Comment by Sparticuz** [comment]
Last commits were Oct 2022, but at this point, it's still ~6 months ago.

**Comment by trim21** [comment]
I hope jesec can grant me permission, but he rejected 😢 So still need to wait for him to review and merge PR.

**Comment by nicjay** [comment]
I see a handful of PRs have been merged in, however it has been over two years since a release. 

@jesec do you have any updates or willing to open up the repository to other maintainers? I ask because I greatly enjoy using Flood and would hate to see it fade in obscurity. 

Thank you.

**Comment by Jogai** [comment]
@jesec relevant question for 2024 too I think ;)

There are quite a few merges done in the past few weeks. Will there be a release?
